### PR TITLE
Draft: V1C-227 docs(mutations): update published pathway definition query

### PR DESCRIPTION
**Pull request by rahulkeerthi**
Changes:
- update publishedPathwayDefinitions query to match recent query changes

relates V1C-227

### CODE REVIEW

File: content/awell-orchestration/api-reference/queries/get-published-pathways.mdx [hunk 1]
```diff
@@ -16,7 +16,7 @@ query GetPublishedPathwayDefinitions {
       id
       title
       version
-      dataPointDefinitions {
+      data_point_definitions {
```
Comment (nckhell) [inline content/awell-orchestration/api-reference/queries/get-published-pathways.mdx:19]: @rahulkeerthi Is this a breaking change or was the documentation incorrect?

File: content/awell-orchestration/api-reference/queries/get-published-pathways.mdx [hunk 2]
```diff
@@ -32,11 +32,27 @@ query GetPublishedPathwayDefinitions {
...
+      track_definitions {
+        id
+        title
+        can_trigger_manually
+      }
+      dataPointDefinitions {
+        # @deprecated use data_point_definitions
+        id
+        # ...
+      }
```
Comment (nckhell) [inline content/awell-orchestration/api-reference/queries/get-published-pathways.mdx:44]: This is confusing and I don't understand what we are saying here.

File: content/awell-orchestration/api-reference/queries/get-published-pathways.mdx [hunk 3]
```diff
@@ -32,11 +32,27 @@ query GetPublishedPathwayDefinitions {
...
     }
   }
 }
 ```
 
+## can_trigger_manually
```
Comment (nckhell) [inline content/awell-orchestration/api-reference/queries/get-published-pathways.mdx:50]:
```suggestion
## Can trigger manually
```

**Review by nckhell — CHANGES_REQUESTED** [review]

File: content/awell-orchestration/api-reference/queries/get-published-pathways.mdx (hunk 3 above)
Comment (rahulkeerthi) [reply content/awell-orchestration/api-reference/queries/get-published-pathways.mdx:50]: This doesn't make sense as a phrase, it's the name of the field this section is about
I can change the header to 'What does can_trigger_manually mean?' or 'The can_trigger_manually field' or 'How can I tell if a track can be added or scheduled to a flow?'

File: content/awell-orchestration/api-reference/queries/get-published-pathways.mdx (hunk 1 above)
Comment (rahulkeerthi) [reply content/awell-orchestration/api-reference/queries/get-published-pathways.mdx:19]: no it is not, you can use either field

File: content/awell-orchestration/api-reference/queries/get-published-pathways.mdx (hunk 2 above)
Comment (nckhell) [reply content/awell-orchestration/api-reference/queries/get-published-pathways.mdx:44]: Make it simple for the user: only document what we expect the user to use moving forward. Given it's not a breaking change, there's no risk of removing the old documentation completely.

Or just something like:
```graphql
{
data_point_definitions { # previously dataPointDefinitions
}

File: content/awell-orchestration/api-reference/queries/get-published-pathways.mdx (hunk 3 above)
Comment (nckhell) [reply content/awell-orchestration/api-reference/queries/get-published-pathways.mdx:50]: Let's do "The can_trigger_manually field"

**Review by nckhell — CHANGES_REQUESTED** [review]

File: content/awell-orchestration/api-reference/queries/get-published-pathways.mdx [hunk 4]
```diff
@@ -1,9 +1,9 @@
 ---
 title: Get published pathway definitions
-description: Get all published pathway definitions in your organization
+description: Get all published pathway definitions for your organization
 ---
 
-Gets the full list of published pathway definitions in your tenant or organization.
+Gets the full list of published pathway definitions (and optionally, track definitions) for your organization.
```
Comment (nckhell) [inline content/awell-orchestration/api-reference/queries/get-published-pathways.mdx:6]:
```suggestion
Retrieve the full list of published pathway definitions including the title, version, baseline data point definitions, and track definitions.
```

**Review by nckhell — APPROVED** [review]

**Pull request merged**